dds/DCPS/FilterEvaluator.cpp

```diff
@@ -24,6 +24,8 @@
 #include <sstream>
 #include <algorithm>
 
+#define MOD "MOD"
```

Comment:
I'd like to avoid preprocessor usage here, it can be an anonymous-namespace-scoped const char[].